bin/src/acme.rs

```diff
@@ -418,30 +418,7 @@ fn order_request(channel: &mut Channel<Request, Response>, request: Request) ->
                 bail!("could not execute request: {}", response.message);
             }
             ResponseStatus::Ok => {
-                // TODO: remove the pattern matching and only display the response message
-                match request.request_type {
-                    Some(RequestType::AddBackend(_)) => {
-                        info!("backend added : {}", response.message)
-                    }
-                    Some(RequestType::RemoveBackend(_)) => {
-                        info!("backend removed : {} ", response.message)
-                    }
-                    Some(RequestType::AddCertificate(_)) => {
-                        info!("certificate added: {}", response.message)
-                    }
-                    Some(RequestType::RemoveCertificate(_)) => {
-                        info!("certificate removed: {}", response.message)
-                    }
-                    Some(RequestType::AddHttpFrontend(_)) => {
-                        info!("front added: {}", response.message)
-                    }
-                    Some(RequestType::RemoveHttpFrontend(_)) => {
-                        info!("front removed: {}", response.message)
-                    }
-                    _ => {
-                        // do nothing for now
-                    }
-                }
+                info!("Request successfull : {}", response.message);
```

Comment:
Could you display the operation kind in the log?

Reply:
I though `response.message` contained a summary of the request, but it turns out it isn't the case.
I'll just revert this change.